js/trash.js

```diff
@@ -145,8 +145,14 @@ class Trashcan {
      * @returns {void}
      */
     updateContainerPosition() {
-        this._container.x = (window.innerWidth / this._scale - Trashcan.TRASHWIDTH) / 2;
-        this._container.y = window.innerHeight / this._scale - Trashcan.TRASHHEIGHT;
+        const offsetPercentageFromBottom = 0.1; // 10% from the bottom
```

Comment:
I think these numbers could be calculated from the size/position of the buttons on the bottom-right.